### ISSUE TITLE
[!] Unable to read the license file `../LICENSE` for the spec `notification_permissions (0.0.1)`

### ISSUE THREAD

**Issue by dpedrinha** [Vanethos/flutter_notification_permissions#18]
When running pod install I'm getting this warnings:

```
[!] A license was specified in podspec `notification_permissions` but the file does not exist - /Users/myuser/dev/apps/myapp/ios/.symlinks/plugins/notification_permissions/LICENSE

[!] Unable to read the license file `../LICENSE` for the spec `notification_permissions (0.0.1)`

[!] Unable to read the license file `../LICENSE` for the spec `notification_permissions (0.0.1)`
```

I checked and the LICENCE.md file is there on the folder. 

I'm using version 0.4.3

**Comment by Vanethos** [comment]
Fixed in v 0.4.4

**Comment by dpedrinha** [comment]
Fix confirmed.
Thank you.
